app/themes/default/views/layouts/default.html.slim

```diff
@@ -33,7 +33,18 @@ html
         div.collapse.navbar-collapse#site-navigation-navbar
           ul.nav.navbar-nav.pull-right
             li
-              = link_to(t('layout.navbar.courses'), courses_path)
+              - my_courses = user_signed_in? && Course.with_approved_user(current_user).ordered_by_end_at
+              - if my_courses.present?
+                a.dropdown-toggle data-toggle="dropdown"
+                  => t('layout.navbar.courses')
+                  span.caret
+                ul.dropdown-menu.pull-right
+                  - my_courses.each do |course|
+                    li = link_to(format_inline_text(course.title), course_path(course))
+                  li.divider role='separator'
+                  li = link_to(t('layout.navbar.all_courses'), courses_path)
+              - else
+                = link_to(t('layout.navbar.courses'), courses_path)
```

Comment:
I'm a little confused. haha. Do you mean the following:
(A) "Course"
(B) "All Courses"   (links to all courses page)
(C) "All Courses"   instead of "Courses"  (links to all courses page)

At the moment, we are not doing a "My Courses" page.